### PR TITLE
docs/pgd/fix/BDR-4594-docs-add-missing-terminology-terms

### PR DESCRIPTION
**Pull request by djw-m**
## What Changed?

Subscription and writer added to the terminology

### CODE REVIEW

**Comment by eatonphil** [comment]
This makes sense to me! I can't comment on the technical accuracy of it though. Perhaps let me cc @pgnickb who jumped into the thread where I asked.

File: product_docs/docs/pgd/5/terminology.mdx [hunk 1]
```diff
@@ -133,3 +137,8 @@ Witness nodes primarily serve to help the cluster establish a consensus. An odd
 #### Write leader
 
 In an Always-On architecture, a node is selected as the correct connection endpoint for applications. This node is called the write leader. Once selected, proxy nodes route queries and updates to it. With only one node receiving writes, unintended multi-node writes can be avoided. The write leader is selected by consensus of a quorum of data nodes. If the write leader becomes unavailable, the data nodes select another node to become write leader. Nodes that aren't the write leader are referred to as *shadow nodes*.
+
+#### Writer
+
+ When a [subscription](#subscription) delivers data changes to a PGD node, the database server tasks a worker process called a writer with getting those changes recorded.
```
Comment (pgnickb) [inline product_docs/docs/pgd/5/terminology.mdx:143]:
```suggestion
 When a [subscription](#subscription) delivers data changes to a PGD node, the database server tasks a worker process called a writer with getting those changes applied.
```

**Comment by pgnickb** [comment]
Lgtm, aside from the small (optional) correction I've added.
Also please have a look at https://www.postgresql.org/docs/current/logical-replication-subscription.html
if you want some inspiration on what to write about `Subscription`.

**Review by pgnickb — APPROVED** [review]

**Comment by djw-m** [comment]
@pgnickb I had a look there but for a terminology page it felt somewhat difficult to meaningfully compress down.

**Pull request merged**